packages/command-utils/lib/index.js

```diff
@@ -22,27 +22,27 @@
  * SOFTWARE.
  */
 
-import fs from 'fs'
-import path from 'path'
-import which from 'which'
-import execa from 'execa'
-import rl from 'readline'
-import chalk from 'chalk'
-
-function info(...args: string[]) {
+const fs = require('fs')
```

Comment:
this needed to be converted to JS because its used by ui-scripts. Also it needs to be in commonJS because its used by commonJS packages like `cz-lerna-changelog`